agent/taskresource/envFiles/envfile.go

```diff
@@ -358,9 +360,14 @@ func (envfile *EnvironmentFileResource) writeEnvFile(writeFunc func(file oswrapp
 	return nil
 }
 
-// Cleanup removes env files downloaded for the task
+// Cleanup removes env file directory for the task
 func (envfile *EnvironmentFileResource) Cleanup() error {
-	// TODO
+	err := envfile.os.RemoveAll(envfile.resourceDir)
+	if err != nil {
+		return fmt.Errorf("unable to remove envfile resource directory %s: %v", envfile.resourceDir, err)
+	}
+
+	seelog.Infof("Removed envfile resource directory at %s", envfile.resourceDir)
```

Comment:
is it possible to add more context such as task arn here ?

Reply:
resource path includes cluster name and task ID, that should be enough identifying information right?

Reply:
ah, it should be good enough